tests/app/v2/notifications/test_post_notifications.py

```diff
@@ -516,24 +371,6 @@ def test_post_email_notification_returns_201_with_celery_persistence(
     assert response.status_code == 201
     resp_json = json.loads(response.get_data(as_text=True))
     assert validate(resp_json, post_email_response) == resp_json
-    assert resp_json["reference"] == reference
-    assert resp_json["content"]["body"] == sample_email_template_with_placeholders.content.replace("((name))", "Bob")
-    assert resp_json["content"]["subject"] == sample_email_template_with_placeholders.subject.replace("((name))", "Bob")
-    assert resp_json["content"]["from_email"] == "{}@{}".format(
-        sample_email_template_with_placeholders.service.email_from,
-        current_app.config["NOTIFY_EMAIL_DOMAIN"],
-    )
-    assert "v2/notifications/{}".format(resp_json["id"]) in resp_json["uri"]
-    assert resp_json["template"]["id"] == str(sample_email_template_with_placeholders.id)
-    assert resp_json["template"]["version"] == sample_email_template_with_placeholders.version
-    assert (
-        "services/{}/templates/{}".format(
-            str(sample_email_template_with_placeholders.service_id),
-            str(sample_email_template_with_placeholders.id),
-        )
-        in resp_json["template"]["uri"]
-    )
-    assert not resp_json["scheduled_for"]
```

Comment:
do we have to delete all this? Seems that we should delete (or move) the other `test_post_email_notification_returns_201()` and keep this one roughly as it is (ie the post to /email doesn't actually put it into the database, but it does return a bunch of stuff like this to the user. Possibly this should be called something like `test_post_email_notification_returns_notification_information()`.

Reply:
skipped